.github/workflows/ci.yaml

```diff
@@ -16,8 +16,8 @@ jobs:
 
     strategy:
       matrix:
-        otp: [26.x]
-        elixir: [1.15.x]
+        otp: [26.x, 27.x]
+        elixir: [1.15.x, 1.16.x, 1.17.x]
```

Comment:
It seems some of these versions are not available with one another; we will need to exclude those from the matrix. See https://github.com/seancribbs/hardhat/actions/runs/9529807072/job/26316538977?pr=28